README.md

```diff
@@ -482,6 +482,18 @@ interface Bot {
 }
 ```
 
+## Observables
```

Comment:
This could be easily confused with reactive observables (see [RxJS](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/observable.md)). Maybe rename this section to Event Emitters?